dwitter/static/js/feed.js

```diff
@@ -1,4 +1,5 @@
-var onDweetChanged = function() {
+function onDweetChanged() {
+  // eslint-disable-next-line newline-per-chained-call
```

Comment:
The Airbnb ESLint config specifies that function call chains with more than 3 calls must be broken into multiple lines. I don't think that's necessary here.